lib/importer/session.js

```diff
@@ -2,7 +2,15 @@ const crypto = require("crypto");
 
 let in_memory_db = new Map();
 
-exports.CreateSession = (request) => {
+class Session {
+  constructor(p = {}) {
+    this.id = p.id;
+    this.filename = p.filename;
+    this.fields = p.fields;
+  }
+}
```

Comment:
Is this object the same thing that @alaric-rd refers to as a "job"? I.e. you can have multiple sessions on the go at once, potentially.

Reply:
On further reading, no, this is what @alaric-rd refers to as a session:

```
 The frontend obtains a session ID, through some appropriate process (eg, asking an application-specific API server that has access to the control interface).
It uploads a file to that session. 
```

The `job` is specific to the mapping so 

```
Each session will contain a single spreadsheet, but may contain many mapping jobs
```